client/src/App.tsx

```diff
@@ -3,16 +3,28 @@ import "./App.css";
 import { AuthContainer } from "./containers";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import PostDashboard from "./containers/PostDashboard";
+import { createTheme } from "@mui/material/styles";
+import ThemeProvider from "@mui/system/ThemeProvider";
+
+const theme = createTheme({
+  palette: {
+    secondary: {
+      main: "#f4003d",
+    },
+  },
+});
 
 function App() {
   return (
+    <ThemeProvider theme={theme} >
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<AuthContainer />} />
         <Route path="/dashboard" element={<PostDashboard/>} /> 
         {/* TODO: Ensure authenticated then redirect to dashboard */}
```

Comment:
In the future, you can add this above the `return`. No need for curly braces that way. It's fine for now though.

Reply:
kk I will keep that in mind